go.mod

```diff
@@ -18,36 +18,35 @@ require (
 	github.com/go-openapi/strfmt v0.21.3
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/golang/mock v1.6.0
-	github.com/onsi/ginkgo/v2 v2.6.0
-	github.com/onsi/gomega v1.27.1
+	github.com/onsi/ginkgo/v2 v2.9.0
+	github.com/onsi/gomega v1.27.2
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.1
 	golang.org/x/crypto v0.7.0
 	golang.org/x/text v0.8.0
-	k8s.io/api v0.25.0
-	k8s.io/apiextensions-apiserver v0.25.0
-	k8s.io/apimachinery v0.25.0
+	k8s.io/api v0.26.1
+	k8s.io/apiextensions-apiserver v0.26.1
+	k8s.io/apimachinery v0.26.1
 	k8s.io/cli-runtime v0.25.0
```

Comment:
do we need to bump this as well?

Reply:
@mkumatag, cluster-api 1.4.x is using v0.26.1 and this change has upgraded these here.

Reply:
I meant for `k8s.io/cli-runtime` which is still pointing to `v0.25.0`

Reply:
> I meant for `k8s.io/cli-runtime` which is still pointing to `v0.25.0`

Oops, missed it. Updated, PTAL!